lib/pxutil.js

```diff
@@ -223,6 +223,19 @@ function sha256(string) {
     return _hashString(string, 'sha256');
 }
 
+function VerifyRegexAndString(match, matchWith) {
```

Comment:
The function name isn't clear to me. How about something like `isStringMatchWith(string, match)`? Then it's a little clearer that the first thing has to be the string we're comparing and the second is some kind of match.

Also I feel like something like this might be clearer too:

```javascript
if (match instanceof RegExp) {
    return string.match(match);
} else if (typeof match === 'string') {
    return string.toLowerCase() === match.toLowerCase();
}
return false;
```

Reply:
you got it